integrationTests/vm/arwen/common.go

```diff
@@ -0,0 +1,10 @@
+package arwen
+
+import (
+	"io/ioutil"
+	"path"
+)
+
+func GetBytecode(relativePath string) ([]byte, error) {
```

Comment:
maybe add mock comment to skip code style warnings. Something like `// GetBytecode --`